.github/workflows/test.yml

```diff
@@ -61,4 +61,5 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-
-      - run: go test -v github.com/obolnetwork/charon/testutil/compose/compose -integration
+      - run: go build # Pre-build current SHA charon binary
```

Comment:
This workaround of pre-built binaries was required since building the binary inside the test code didn't work